lib/util/location.ex

```diff
@@ -97,6 +99,85 @@ defmodule Util.Location do
     distance(as_location!(lhs), as_location!(rhs))
   end
 
+  @doc """
+  Returns the displacement from `from` to `to` as a `Util.Vector2d` given in meters.
+
+  The `x` direction is east, and the `y` direction is north.
+
+  ## Examples
+      iex> Util.Location.displacement(
+      ...>    Util.Location.new(42, -71.0002),
+      ...>    Util.Location.new(42, -71.0001)
+      ...> )
+      %Util.Vector2d{x: 8.263404849676181, y: 0.0}
+
+      iex> Util.Location.displacement(
+      ...>    Util.Location.new(42, -71.0002),
+      ...>    Util.Location.new(42.0001, -71.0002)
+      ...> )
+      %Util.Vector2d{x: 0.0, y: 11.119508023696506}
+
+  If the displacement is south and/or west, then the `x` or `y` coordinate will be negative
+
+  ## Example
+      iex> Util.Location.displacement(
+      ...>    Util.Location.new(42.0002, -71.0001),
+      ...>    Util.Location.new(42.0001, -71.0002)
+      ...> )
+      %Util.Vector2d{x: -8.26337887771932, y: -11.119508022906418}
+  """
+  @spec displacement(from :: __MODULE__.From.t(), to :: __MODULE__.From.t()) :: Vector2d.t()
+  def displacement(%__MODULE__{latitude: from_latitude, longitude: from_longitude}, %__MODULE__{
```

Comment:
just a note, I'm having difficulty understanding this in context. I understand what it's doing, but when I read `segment_vector = displacement(start_loc, end_loc)` it just doesn't make sense to me from reading it, and I've gotta do extra "mental 'math'" to figure out what I'm supposed to understand. Do you think there could be a different word to accurately describe this, or a way I should be thinking about this word?

Reply:
Let's ideate tomorrow. I've been steeped in this for a while now, so it makes probably _too much sense_ to me 😅

Reply:
I'm thinking maybe... `displacement_from`, and swap the order of the arguments. That way it would be something like:

```elixir
end_loc |> displacement_from(start_loc)
```

And then a similar treatment to `displace`, so that it would become

```elixir
start_loc |> displace_by(vector)
```

(I know that credo won't let us do it _exactly_ like that, but it still might make it easier to tell what's going on.)

Reply:
I like the idea of reordering the arguments 👍🏻

Reply:
[How does this look?](https://github.com/mbta/skate/pull/2510/commits/7bfb9a68b22a5daa540ba86ca751548cf115595a)

Reply:
[And this to fix some formatting](https://github.com/mbta/skate/pull/2510/commits/4c7295e35b67940a1d354a80b9a6e4bd7cbc4bed)